server.bat

```diff
@@ -0,0 +1 @@
+python server.py
```

Comment:
I don't think we need to run a server here, but let me know otherwise. I'm planning on removing them.